content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md

```diff
@@ -0,0 +1,379 @@
+---
+db_id: 
```

Comment:
leave `db_id` out. For two reasons:
1. `db_id` is not a thing. `_db_id` is a thing. I dont believe this would pass the linter (have you run the linter?)
2. `_db_id` is filled in automatically as stuff is loaded into the db. So you dont need to write this